src/pages/post/index.tsx

```diff
@@ -20,53 +20,52 @@ import { useRouter } from 'next/router'
 import type { ReactElement, ChangeEventHandler } from 'react'
 import { useState } from 'react'
 import { useCreateUploadImagesMutation } from '@apis/image'
+import type { CreatePostReq } from '@apis/post'
 import { useGetCategoriesQuery, useCreatePostMutation } from '@apis/post'
 import { localeCurrencyToNumber } from '@utils/format'
 import { PostForm } from '@components'
 import { TRADE_TYPES, PRODUCT_CONDITIONS } from '@constants'
 import { useResponsive } from '@hooks'
-import type { ProductConditionCodes, TradeTypeCodes } from '@types'
-
-type PostFormStatus = {
-  title: string
-  imageInfos: ImageInfo[]
-  category: string
-  price: string
-  location: string
-  productCondition: ProductConditionCodes | ''
-  tradeType: TradeTypeCodes | ''
-  description: string
+
+type PostFormStatus = Partial<
+  Pick<
+    CreatePostReq,
+    | 'title'
+    | 'description'
+    | 'location'
+    | 'category'
+    | 'tradeType'
+    | 'productCondition'
+  >
+> & {
+  price?: string
+  imageInfos?: ImageInfo[]
 }
 
 type HandleUpdatePostForm = ChangeEventHandler<
   HTMLTextAreaElement | HTMLInputElement | HTMLFormElement
 >
-const initialPostForm: PostFormStatus = {
-  imageInfos: [],
-  title: '',
-  category: '',
-  location: '',
-  productCondition: '',
-  tradeType: '',
-  description: '',
-  price: ''
-}
+
+const checkCompleteForm = (
```

Comment:
```suggestion
const isCompleteForm = (
```
boolean을 리턴하는 type predicate는 앞에 is prefix가 붙으면 좋을 것 같습니다!

Reply:
[반완](https://github.com/price-offer/offer-fe/pull/179/commits/ece4fd4bc0136852f309f5a0d0942e22da238ba1#diff-9d02a591eb4f45224eb30038892cdc150e288603114267278ec08316fad10444R49)